certified-connectors/Tesseron Invoice/apiDefinition.swagger.json

```diff
@@ -217,6 +217,408 @@
+                            "properties": {
+                              "ServiceAssignmentTypeId": {
+                                "type": "integer",
+                                "Title": "Type ID"
+                              },
+                              "AssignmentTypeName": {
+                                "type": "string",
+                                "title": "Type Name"
+                              }
+                            }
+                          },
+                          "EnterpriseId": {
+                            "type": "integer",
+                            "title": "Enterprise ID",
+                            "description": "The enterprise ID."
+                          },
+                          "EnterpriseName": {
+                            "type": "string",
+                            "title": "Enterprise Name",
+                            "description": "The name of the enterprise."
+                          },
+                          "TicketId": {
+                            "type": "integer",
+                            "title": "Ticket ID",
+                            "description": "The ticket ID."
+                          },
+                          "Engineer": {
+                            "type": "object",
+                            "title": "Engineer",
+                            "description": "Details of the engineer",
+                            "properties": {
+                              "EngineerId": {
+                                "type": "integer",
+                                "title": "Engineer ID",
+                                "description": "The ID of the engineer."
+                              },
+                              "EngineerName": {
+                                "type": "string",
+                                "title": "Engineer Name",
+                                "description": "The name of the engineer."
+                              },
+                              "UserID": {
+                                "type": "integer",
+                                "title": "Engineer User ID",
+                                "description": "The user name of the engineer."
+                              },
+                              "ResourceNumber": {
+                                "type": "string",
+                                "title": "Engineer Resource Number",
+                                "description": "The resource number of the engineer."
+                              }
+                            }
+                          },
+                          "Ticket": {
+                            "type": "object",
+                            "properties": {
+                              "ticketId": {
+                                "type": "integer",
+                                "title": "Ticket ID",
+                                "decription": "The ticket ID."
```

Comment:
The property 'decription' is not valid. If this property is an extension, its name must begin with "x-". ValuePath: /paths/~1GetServiceAssignmentsDispatcher/post/responses/200/schema/properties/Result/properties/Assignments/items/properties/Ticket/properties/ticketId/decription

Reply:
fixes spelling mistake